docs/Project-Configuration/Task-Priority.md

```diff
@@ -0,0 +1,31 @@
+# Task Priority
+
+Evergreen orders its task queues by taking into account a number of properties
+of a task. These properties are given weights and are summed. Evergreen orders
+the queues according to this sum. Weights are editable by Evergreen admins.
```

Comment:
"Weights are editable by Evergreen admins."
Where is this editable?

Reply:
In the scheduler section of the admin page. I didn't include it, since this documentation is user-facing.

Reply:
Yeah, this is just for my knowledge, thanks!